### PR TITLE
Fix the survey URL used on experiment page

### LINKED ISSUE #748: Fix `Give Feedback` buttons on experiments page.
#726 introduced a few small errors:

~~https://testpilot.stage.mozaws.net/experiments/activity-stream~~ (enabled view)

/edit stage has been rolled back. You can verify on [dev](http://testpilot.dev.mozaws.net/experiments/activity-stream).
- Doesn't open in a new tab.
- Survey URL is malformed (https://qsurvey.mozilla.com/s3/activity-stream?ref=givefeedback&experiment=Activity%20Stream?ref=givefeedback&experiment=Activity%20Stream&installed=%40activity-streams&installed=universal-search%40mozilla.com).
- ~~"Give Feedback" doesn't have hover pointer.~~

### PR DESCRIPTION
**Pull request by lmorchard**
- Refactor survey URL building into experiment model
- Remove redundant survey URL manipulation
- Test to expose the issue

Fixes #748

### CODE REVIEW

File: testpilot/frontend/static-src/app/views/experiment-page.js [hunk 1]
```diff
@@ -310,16 +300,11 @@ export default PageView.extend({
...
-    const queryParams = querystring.stringify({ref: 'givefeedback',
-                                               experiment: this.model.title,
-                                               installed: installed});
-
     app.sendToGA('event', {
       eventCategory: 'ExperimentDetailsPage Interactions',
       eventAction: 'button click',
       eventLabel: 'Give Feedback',
-      outboundURL: `${this.model.survey_url}?${queryParams}`
+      outboundURL: this.model.buildSurveyURL('givefeedback')
```
Comment (ckprice) [inline testpilot/frontend/static-src/app/views/experiment-page.js]: Surveys should open in a new tab.

For reference, [this is how the `experiment-page.js` interaction](https://github.com/mozilla/testpilot/pull/735/commits/f90354eac591ab998369eea52ed42520a5478857) used to work before it was over-written by #726. Just used regular `href` and `target=_blank` then sent the GA event separately.

Comment (meandavejustice) [inline testpilot/frontend/static-src/app/views/experiment-page.js]: just need to add `newTab: true` to our options object here

Comment (ckprice) [inline testpilot/frontend/static-src/app/views/experiment-page.js]: The `newTab` and corresponding `window.open` usage was removed because of #711.

**Comment by lmorchard** [comment]
Added the code to open in a new tab

**Comment by johngruen** [comment]
@lmorchard throwing a test error, but the ux looks fine to me.

**Comment by lmorchard** [comment]
Ugh, okay, think I fixed the tests

**Pull request merged**